.lycheeignore

```diff
@@ -6,9 +6,14 @@ https://support.arduino.cc/hc/*
 
 # Returns 404 for some reason
 https://create.arduino.cc/iot/*
-https://cloud.arduino.cc/home/registration
 https://cloud.arduino.cc/home/resources
 https://cloud.arduino.cc/plans
+https://app.arduino.cc/resources
+https://app.arduino.cc/devices 
+https://app.arduino.cc/things 
```

Comment:
I think these trailing spaces may be preventing the URLs from being filtered correctly:

```suggestion
https://app.arduino.cc/devices
https://app.arduino.cc/things
```